vrecord

```diff
@@ -350,7 +350,7 @@ _setup_vrecord_process(){
         fi
     }
     # set up input and playback
-    unset GRAB_INPUT
+    GRAB_INPUT=()
     GRAB_INPUT+=("${EXTRAINPUTOPTIONS[@]}")
```

Comment:
Would `GRAB_INPUT=("${EXTRAINPUTOPTIONS[@]}")` nor be the same, but shorter for both lines?

Reply:
That would be the same and shorter.